tdd/blackjack/hand.rb

```diff
@@ -0,0 +1,19 @@
+require './deck'
+class Hand
+  attr_accessor :cards
+  def initialize(deck)
```

Comment:
Better to have hand just get cards instead of a deck as the argument. That way the hand model won't break if for some reason the #deal method is renamed. Likewise with the Deck#hit method - just feed it a card.